WordPress/Classes/Utility/ContextManager.m

```diff
@@ -132,6 +139,80 @@ - (void)performAndSaveUsingBlock:(void (^)(NSManagedObjectContext *context))aBlo
     }];
 }
 
+#pragma mark - Store Changes
+
+- (void)observeStoreChanges
+{
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(handleStoreChanges:)
+                                                 name:NSPersistentStoreRemoteChangeNotification
+                                               object:self.persistentContainer.persistentStoreCoordinator];
+}
+
+- (void)handleStoreChanges:(NSNotification *)notification
+{
+    __weak __typeof(self) weakSelf = self;
+    [self.remoteChangeQueue addOperationWithBlock:^{
+        [weakSelf processRemoteChanges];
+    }];
+}
+
+- (void)processRemoteChanges
```

Comment:
Could we document this function a bit? I think it would help as it is not very straightforward (at least to me).

Reply:
Sure, I can add some comments to it.

Reply:
Added! Let me know if it's still unclear, I can add more if needed.